src/platform.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#if defined(_MSC_VER)
+#   define CATA_COMPILER_MSVC
+#endif
+
+#if defined(_WIN32) || defined(_WIN64)
+#   define CATA_OS_WINDOWS
+#
+#   if !defined(WIN32_LEAN_AND_MEAN)
+#       define WIN32_LEAN_AND_MEAN
+#   endif
+#
+#   if !defined(NOMINMAX)
+#       define NOMINMAX
+#   endif
+#
+#   if !defined(STRICT)
+#       define STRICT
+#   endif
+#
+#   include <SdkDdkVer.h>
```

Comment:
[SdkDdkVer.h](http://msdn.microsoft.com/en-us/library/windows/desktop/aa383745%28v=vs.85%29.aspx) defines symbolic constants for the values used for the OS selection macros; nothing more, nothing less. i.e NTDDI_WINXPSP3 instead of 0x05010300.